README.md

```diff
@@ -183,6 +183,7 @@ You can use `CPM_SOURCE_CACHE` on GitHub Actions workflows [cache](https://githu
 If set, CPM will forward all calls to `CPMFindPackage` as `CPMAddPackage`.
 This is useful to create reproducible builds or to determine if the source parameters have all been set correctly.
 This can also be set as an environmental variable.
+This can be controlled on a per project basis with the `CPM_<dependency name>_DOWNLOAD` variable.
```

Comment:
I didn't see a place that listed contributors, so I added NVIDIA here.
If you don't want I can drop

Reply:
Removed since the project is looking at a better way to list contributors.